mkt/ratings/helpers.py

```diff
@@ -13,15 +12,7 @@ def stars(num, large=False):
         return _('Not yet reviewed')
     else:
         num = min(5, int(round(num)))
-        t = jingo.env.get_template('ratings/reviews_rating.html')
-        # These are getting renamed for contextual sense in the template.
-        return jinja2.Markup(t.render({'rating': num, 'detailpage': large}))
-
-
-@jingo.register.function
-def impala_reviews_link(addon):
-    t = jingo.env.get_template('ratings/reviews_link.html')
-    return jinja2.Markup(t.render({'addon': addon}))
+        return _('Rated %s out of 5 stars' % num)
```

Comment:
One says "Not yet reviewed" and the other says "Rated...". I wonder if we should be consistent in calling them ratings or reviews?